### PR TITLE
Create new comment input with button

### PR DESCRIPTION
**Pull request by sergous**
https://github.com/dtpstat/dtp-map/issues/82

Снимок экрана 2021-12-13 в 1 42 16 AM

### CODE REVIEW

**Review by kachkaev — APPROVED** [review]

**Comment by kachkaev** [comment]
Bike-shed mode: which PR & commit style do we want to use? I’ve been sticking with imperative in the past, see https://stackoverflow.com/a/3580764/1818285.

File: src/components/text-input.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledInput = styled.input`
```
Comment (kachkaev) [inline src/components/text-input.tsx:4]: We might need to switch to `textarea` in the future for multi-line comments. Another thing to think about is autoparsing links when displaying comments (e.g. using [`linkifyjs`](https://github.com/Hypercontext/linkifyjs) via [`linkify-react`](https://github.com/Hypercontext/linkifyjs/tree/main/packages/linkify-react). That can be done once we’ve created a full data cycle for single-line comments first 🙂

**Comment by sergous** [comment]
I'll create this 2 as separate tasks for next sprint.

пн, 13 дек. 2021 г., 07:33 Alexander Kachkaev ***@***.***>:

**Comment by sergous** [comment]
> `<BikeShedding>` Which PR & commit style do we want to use? I’ve been sticking with imperative in the past, see https://stackoverflow.com/a/3580764/1818285. This is the most common choice in the community, but we don’t have to follow it. `</BikeShedding>`

I'm OK with both options. Directive style it is then. Please add codestyle page to the wiki for the rest of the team. @kachkaev

**Comment by kachkaev** [comment]
Right, let’s go for Imperative comment and PR style then (as if we are writing a food recipe). We can write down this convention to the wiki later, but because the team is quite small at this point, we can prioritise [**individuals and interactions over processes and tools** and **working software over comprehensive documentation**](https://agilemanifesto.org/) 🙂

Ideally, our coding rules should be expressed as CI checks. This eliminates a need to have too much text in the wiki, which tends to get out of sync with the reality all the time. It’s easy to ignore a document, but it‘s quite hard to ignore ❌  in a PR 😁 If a rule is not expressible as a CI check, it is likely unimportant or too hard to formalise (so probably wrong).

**Comment by sergous** [comment]
If the rules inside CI, that is good. But if they are ONLY inside CI, the question is what is the procedure of making and updating them?

**Pull request merged**